module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -101,20 +102,38 @@ std::string ServiceDesktop::prepareBackupFilename()
     return std::string(backupFileName.data());
 }
 
+std::string ServiceDesktop::prepareSyncFilename()
+{
+    std::array<char, 64> syncFileName{};
```

Comment:
magic number?

Reply:
changed to a const variable with meaningful name